src/main/assets/js/upload-manager/index.ts

```diff
@@ -0,0 +1,64 @@
+import Uppy from '@uppy/core';
+import DropTarget from '@uppy/drop-target';
+import FileInput from '@uppy/file-input';
+import ProgressBar from '@uppy/progress-bar';
+import XHRUpload from '@uppy/xhr-upload';
+
+import { DOCUMENT_MANAGER, UPLOAD_YOUR_DOCUMENTS } from '../../../steps/urls';
+import { getById } from '../selectors';
+
+import { onError, onFilesSelected } from './events';
+import { updateFileList } from './updateFileList';
+import { UploadedDocuments } from './uploadedDocuments';
+
+import '@uppy/drop-target/src/style.scss';
+import '@uppy/progress-bar/src/style.scss';
+
+((): void => {
+  if (document.location.pathname !== UPLOAD_YOUR_DOCUMENTS) {
```

Comment:
I would rather do this jquery style and detect an element on the page that sort of triggers this stuff

Reply:
👍 Good idea, I've swapped this out to check if the element is on the page rather than checking the URL